src/sentry/web/frontend/oauth_token.py

```diff
@@ -86,8 +86,7 @@ def _get_open_id_token(self, grant, request):
             open_id_token = OpenIDToken(
                 request.POST.get("client_id"),
                 grant.user_id,
-                # Encrypt with a random secret until we implement secure shared secrets in prod
-                secrets.token_urlsafe(),
+                options.get("codecov.client-secret"),
```

Comment:
If the option isn't set it returns the default value of the option, which is probably the empty string.

Reply:
@mdtro should we check and if it its an empty string throw some sort of internal error?

Reply:
@EricHasegawa Yes, if the value is not set we should error out and not sign tokens.

Is it also possible to rename the option to something else to avoid confusion? `client-secret` doesn't quite align here and could be confused with as us acting as an OAuth client to Codecov? Something like `codecov.shared-signing-secret` would make more sense to me.

Reply:
what do you think of the approach taken here where we just leave out the id token if we cant get the `option`? This should never happen really since it's statically set in prod, but if it does we can just ignore it - if you want to throw an error, it would have to be a `5xx` which seems weird to throw deliberately.

Reply:
@mdtro bump

Reply:
Omitting the token is good. 👍